templates/feed.php

```diff
@@ -65,4 +51,4 @@
 		<?php endwhile; ?>
 	<?php endif; ?>
 </channel>
-</rss><?php exit; ?>
\ No newline at end of file
+</rss><?php exit; ?>
```

Comment:
Not sure TBH. Lets leave it for now - there might be a reason ;) No need to change it uneccessarily